### ISSUE TITLE
Problems with timestamp and using now() as the default value. 

### ISSUE THREAD

**Issue by goggi** [directus/next#742]
If I set the default value of a timestamp (datetime) to now(), this gets exchanged with the current timestamp. The scenario is that I add data to the database without going through the directus API and need the created_date to be populated. It works if I go to the database and add now() as default value, but as soon as I edit the field in directus the now() value is exchanged to the current timestamp. So as long as I don't edit the field in directus it will keep the now() value as default. 

![image](https://user-images.githubusercontent.com/1946339/96700339-9ff7cb80-138f-11eb-93f2-5293ea349e09.png)

After save: 
![image](https://user-images.githubusercontent.com/1946339/96700384-ae45e780-138f-11eb-8f41-8307031a2320.png)

---

### Project details
Directus Version: 9.0.0-rc.1
Environment: production
OS: Linux 4.15.0-88-generic
Node: 14.7.0
Database: Postgres

**Comment by Nitwel** [comment]
I don't think we have a feature that allows you to set `now()` as a default?
And I don't see a use case for having something like this because there are already the `on create` and `on update` options.
What exactly do you wan't to archive?

**Comment by goggi** [comment]
The scenario is that I add data to the database without going through the directus API and need the created_date to be populated. So I saved data directly into the database. The created_date is not set as its something that happens in the API. But what would be great is that if I set the default value directly inside the database to now() the value is left like that even if I make adjustments to the field inside the admin app. Right now the now() value will be replaced with the current timestamp if I make any adjustments to the field inside the admin app.
